main.go

```diff
@@ -36,12 +36,18 @@ func main() {
 	listenAddress := flags.Flag("listen", "Address on which to expose metrics.").String()
 	whoopingAddress := flags.Flag("whooping", "Address on which to start whooping.").String()
 	tcpCheckerAddresses := flags.Flag("tcp-checker", "TCP address to check. Supports <address>:<port> or <name>:<address>:<port>").Strings()
+	enableReinitiator := flags.Flag("enable-reinitiator", "Enables re-initiation of connections when expected Security Associations are missing").Bool()
 	log.AddFlags(flags)
 	flags.HelpFlag.Short('h')
 	flags.Version(version)
 	socket := flags.Flag("vici-socket", "VICI (charon.vici) socket to connect to. Usually /var/run/charon.vici").String()
 	kingpin.MustParse(flags.Parse(os.Args[1:]))
 
+	if *enableReinitiator && *socket == "" {
```

Comment:
Just to be consitent can we use either `len(str) == 0` or `str == ""`. In line 148 we use the former. :) 
```suggestion
	if *enableReinitiator && len(*socket) == 0 {
```

Reply:
👍